src/client/src/components/SelectableCard/styles.module.css

```diff
@@ -161,3 +161,8 @@
   display: flex;
   font-size: 14px;
 }
+
+.disabledOverlay {
```

Comment:
Shouldn't there just be a style for `button:disabled`

Reply:
Perfectly makes sense.